Wikipedia/Code/WMFNearbyContentSource.m

```diff
@@ -102,6 +119,48 @@ - (void)loadNewContentForce:(BOOL)force completion:(nullable dispatch_block_t)co
 
 - (void)removeAllContent {
     [self.contentStore removeAllContentGroupsOfKind:WMFContentGroupKindLocation];
+    [self.contentStore removeAllContentGroupsOfKind:WMFContentGroupKindLocationPlaceholder];
+}
+
+- (void)showAuthorizationPlaceholder:(nonnull dispatch_block_t)completion {
+    [self.contentStore removeAllContentGroupsOfKind:WMFContentGroupKindLocation];
+    NSURL *placeholderURL = [WMFContentGroup locationPlaceholderContentGroupURL];
+    NSDate *date = [NSDate date];
+    // Check for group for date to re-use the same group if it was updated today
+    WMFContentGroup *group = [self.contentStore firstGroupOfKind:WMFContentGroupKindLocationPlaceholder];
+
+    if (group && (group.wasDismissed || [group.midnightUTCDate isEqualToDate:date.wmf_midnightUTCDateFromLocalDate])) {
+        completion();
+        return;
+    }
+    // If the group doesn't exist (or it doesn't exist for today) - Pick a new random article to show
+    CLLocationCoordinate2D center = CLLocationCoordinate2DMake(48.86611, 2.31444);
```

Comment:
It's the location of Petit Palais, from the mockup: https://en.wikipedia.org/wiki/Petit_Palais 

This is all going away, the plan is to replace it with a static curated list.